service-commands/src/commands/service-commands.json

```diff
@@ -205,6 +208,11 @@
       "node libs/initScripts/local.js distribute"
     ]
   },
+  "account": {
+    "up": [
```

Comment:
might be worth naming this a bit more clearly like just `accounts` - `print`

Reply:
I left this as is b/c `up` is used in all the other scripts and I didn't know if I should add another field to the enum here https://github.com/AudiusProject/audius-protocol/blob/8d8333e2c548a8d57d805824bb6e101984f2966d/service-commands/src/setup.js#L104-L114

Reply:
eh yeah, its all good